app/v2/services/RetrieveCrystallisationObligationsService.scala

```diff
@@ -36,15 +36,15 @@ class RetrieveCrystallisationObligationsService @Inject()(connector: RetrieveCry
       ec: ExecutionContext): Future[ServiceOutcome[RetrieveCrystallisationObligationsResponse]] = {
 
     val result = for {
-      downstreamResponseWrapper <- EitherT(connector.retrieveCrystallisationObligations(request)).leftMap(mapDownstreamErrors(downstreamErrorMap))
+      downstreamResponseWrapper <- EitherT(connector.retrieveCrystallisationObligations(request)).leftMap(mapDownstreamErrors(errorMap))
       mtdResponseWrapper        <- EitherT.fromEither[Future](filterCrystallisationValues(downstreamResponseWrapper))
     } yield mtdResponseWrapper
 
     result.value
 
   }
 
-  private val downstreamErrorMap: Map[String, MtdError] =
+  private val errorMap: Map[String, MtdError] =
```

Comment:
Is there a specific reason the name was changed here? We usually call this val downstreamErrorMap in other API endpoints.

Reply:
They're all meant to be called errorMap.